{
  try {
    return processNext(event);
  }
 catch (  Exception e) {
    e=(Exception)ExceptionHelper.sanitizeIfNeeded(e);
    return event.getFlowConstruct().getExceptionListener().handleException(e,event);
  }
}
